src/components/attendee/Button/Button.tsx

```diff
@@ -1,11 +1,35 @@
+import { ButtonHTMLAttributes } from "react";
+import classNames from "classnames";
+
 import CN from "./Button.module.scss";
 
-type ButtonProps = {
+enum ButtonVariant {
+  primary = "primary",
+  alternative = "alternative",
+  alternativeBorder = "alternativeBorder",
+}
```

Comment:
I don't think there is a need for this. Just an alias is enough:

`type ButtonVariant = "primary" | "alternative" | "alternativeBorder";`

This way it will be used in two places. And the check down can just be made with the string itself `variant === "primary"`.

There are yet other ways of doing this, but I think for the purpose here, what I suggest is KISS enough

Reply:
done, [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)